File: Code/dotNetBasics/ConsoleApp/ADStack/Components/PathPlanner.cs
using ConsoleApp.ADStack.Models;

namespace ConsoleApp.ADStack.Components
{
    internal class PathPlanner : IComponent
    {
        public void Start()
        {
            Console.WriteLine("[PathPlanner] Starting.");
        }

        public void Stop()
        {
            Console.WriteLine("[PathPlanner] Stopping.");
        }
    }
}

File: Code/Fluent.Java/src/adstack/components/PathPlanner.java
package adstack.components;

import adstack.models.IComponent;

public class PathPlanner implements IComponent {

	@Override
	public void start() {
        System.out.println("[PathPlanner] Starting.");
	}

	@Override
	public void stop() {
        System.out.println("[PathPlanner] Stopping.");
	}

}
